src/blocks/modal.css

```diff
@@ -38,17 +38,17 @@
 }
 
 .modal__close {
-  background-image: url("../images/close-button.svg");
-  position: absolute;
-  bottom: calc(100% + 5px);
-  left: calc(100% + 5px);
+  background-image: url(../images/close_icon.svg);
```

Comment:
` url(../images/close_icon.svg);`

The project won't start. See the errors https://disk.yandex.ru/i/_-eSnW2OXtwS-g 


You had the correct styles last time. You shouldn't have changed it



```js

  background-image: url("../images/close-button.svg");

```